build.xml

```diff
@@ -1278,13 +1278,12 @@
       <condition property="maxMemory" value="8G" else="4G">
           <equals arg1="${test.classlistprefix}" arg2="distributed"/>
       </condition>
-      <testmacrohelper inputdir="${test.dir}/${test.classlistprefix}" filelist="@{test.file.list}" exclude="**/*.java" timeout="${test.timeout}">
+      <testmacrohelper inputdir="${test.dir}/${test.classlistprefix}" filelist="@{test.file.list}" exclude="**/*.java" timeout="${test.timeout}" maxmemory="${maxMemory}">
```

Comment:
the new epoch sync test was failing and turns out we try to set Xmx below, but that is override by `testmacrohelper` who uses `1024m`.  Need to pass `maxMemory` along to avoid that...

there is a thread about this in https://the-asf.slack.com/archives/CK23JSY2K/p1718991597495909

Accord branch has the `maxmemory` as a property we can override, but not on any other branch...